src/OrchardCore/OrchardCore.Media.Abstractions/Events/IMediaEventHandler.cs

```diff
@@ -14,4 +14,10 @@ public interface IMediaEventHandler
         Task MediaMovingAsync(MediaMoveContext context);
         Task MediaMovedAsync(MediaMoveContext context);
     }
+
+    public interface IMediaEventHandler2 : IMediaEventHandler
```

Comment:
Why is a second interface needed?

Reply:
That was just a quick way to avoid breaking changes in the `IMediaEventHandler` interface. Even if I don't really like them, do you use default interface methods in Orchard for those changes?

Reply:
From the top of my head I don't know about any use cases, but usually, we have base classes for such interfaces to avoid such breaking changes. There's indeed `MediaEventHandlerBase` where you can add a new method for this.

Reply:
So I should just add to the existing interface? I've already extended the base class...

Reply:
Yep.

Reply:
Ok, done.

Reply:
Great!